build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath deps.gradle_plugins.android
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
There's no need to remove variable for the build gradle tools dependency with a hardcoded dependency.

Reply:
This was automatically done by Android Studio, I did not touch it.

Reply:
Uhm guess Android Studio didn't know what it was doing there. Avoid committing changes, which have nothing to do with the main change. It should only touch the necessary lines/ files for a specific change.